sdk/spring/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Release History
 
 ## 4.0.0-beta.4 (Unreleased)
+Please refer to [Spring Cloud Azure Migration Guide for 4.0](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.3/4.0.0-beta.3/reference/html/appendix.html#migration-guide-for-4-0) to learn how to migrate to version 4.0.
```

Comment:
I prefer to delete this in current PR. Because it's not related to your current task.
Even if you want to add it, the version in link should be `beta.4` instead of `beta.3`.

Reply:
will delete it